include/service/entities/plugin_service.js

```diff
@@ -1216,7 +1216,7 @@ module.exports = function PluginServiceModule(pb) {
                              pb.log.debug('PluginService:[%s] Registering localizations for locale [%s]', details.uid, locale);
                          }
 
-                         var result = pb.Localization.registerLocalizations(locale, localizations[locale]);
+                         var result = pb.Localization.registerLocalizations(locale, localizations[locale], plugin.dirName);
```

Comment:
Please use ```details.uid``` here.  There are instances where the folder name does not match the plugin UID.  We are trying to shy away from that and deprecate that functionality.